boards/arm/rak5010_nrf52840/rak5010_nrf52840.yaml

```diff
@@ -0,0 +1,20 @@
+identifier: rak5010_nrf52840
+name: RAK5010-NRF52840
+type: mcu
+arch: arm
+flash: 256
```

Comment:
Might be missing sthg but i think the flash and ram values need to be swapped.

Reply:
Ok I'll check